globalbibletools/semantic-dictionary
# English Data (Hebrew and Greek)

What's Changed:
- Changed the script to take the original language (hebrew or greek) as an additional argument
  - Modified error messages around this change.
- Filled `data/eng` with data from:
  - [UBS Hebrew Dictionary Version 0.9.1 en ](https://github.com/ubsicap/ubs-open-license/blob/main/dictionaries/hebrew/XML/UBSHebrewDic-v0.9.1-en.XML)
  - [UBS Greek NT Dictionary Version 1.0 en](https://github.com/ubsicap/ubs-open-license/blob/main/dictionaries/greek/XML/UBSGreekNTDic-v1.0-en.XML)

QA:
- Look at eng data to make sure it is correct
  - Note: all of the xml UBS Greek NT dictionaries contain comments spread about in English and Traditional Mandarin. These did not interfere with the structure of the xml.
- Test the script to ensure it outputs the correct error messages.
